### DISCUSSION TITLE
Welcome to MemTorch Discussions!

### DISCUSSION THREAD

**Discussion by coreylammie** [coreylammie/MemTorch#97] [General]
MemTorch is now using GitHub Discussions instead of `Gitter` for general discussions surrounding the MemTorch simulation framework and topics related to Memristive Deep Learning Systems. We intend for discussions to act in a complementary manner to Issues and Feature Requests accessible using the `Issues` tab.

Using GitHub discussions, you are welcome and encouraged to:

Ask questions you’re wondering about.
Share ideas.
Engage with other community members.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by McLemoreT** [comment]
I am Tyler McLemore. I've only been using the software for a few weeks but find it very interesting. I was introduced to it through my university, and now I am using it for some of my clinical work. I love your work and appreciate all the effort you have put in.

**Comment by Agenttuo** [comment]
Hello, I have a question for you. The accuracy rate of mobilenet v2 memristive neural network on cifar10 is only more than 20%. Is this normal? Finally thank you for your contribution.

**Comment by xiaoqunLiu** [comment]
Hello, I got a question. What is the corresponding version of torch correspond with memtorch 1.1.6 please?

**Comment by spectacles9468** [comment]
How can I use it in colab because I can't install it there? Has anybody got an suggestions (other than updating sklearn to scikit) on how to do that?

**Comment by BehnazNekoonam** [comment]
Hi dear Corey, My name is Behnaz and I'm studying neuromorphic computing. I want to train my memristive neural network  with being aware of memristor's conductance constraints. 
I'll be happy to join this community and ask my questions!

**Comment by sneha123rt** [comment]
I'm encountering a `ModuleNotFoundError` for `memtorch_bindings`. There is no documentation available regarding this module. Could you please provide guidance on how to resolve this issue? Thank you.

**Comment by chipher-zero** [comment]
Hi Corey, I am currently working on a project modelling multi-state CBRAM devices using physics informed neural networks (still work in progress, but upon request I am more than happy to share some results).

I am studying the memtorch framework, and it seems like an excellent starting point for simulating the performance of memristors on real-world tasks. My question is: Is it possible to include, say our CBRAM model on the memtorch framework? If yes, were is it a good place to start?
